pysimt/models/snmt_rnn.py

```diff
@@ -215,7 +215,7 @@ def setup(self, is_train=True):
     def load_data(self, split, batch_size, mode='train'):
         """Loads the requested dataset split."""
         # For wait_if_diff, wait_if_worse and test-time waitk decodings
-        if mode == 'beam' and self.opts.model['translator_type'] != 'gs':
+        if mode == 'beam' and self.opts.model['translator_type'] != 'gs' and self.opts.model['translator_type'] != 'bs':
```

Comment:
you can combine the last two checks: `self.opts.model['translator_type'] not in ('gs', 'bs')`